docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -365,7 +365,21 @@ Continue configuring your Kubernetes provider by clicking **Add region** and com
       runAsGroup: 10001
     ```
 
-    <br>Note that you cannot change users during the Helm upgrades.
+    Note that you cannot change users during the Helm upgrades.
+
+  - Overrides to use a secret:
+
+    ```yml
+    tserver:
+      secretEnv:
+      - name: {{ENV-NAME-IN-TSERVER}}
+        valueFrom:
+          secretKeyRef:
+            key: <KEY NAME>
+            name: <SECRET NAME>
+    ```
+
+    Note that this override only works with user-created namespaces, not platform-created namespaces.
```

Comment:
`s/platform/YBA`?

Also, can we link to https://docs.yugabyte.com/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/#configure-the-cloud-provider:~:text=Use%20the%20Namespace%20field%20to%20specify%20the%20namespace to clarify what we mean by a user-created namespace. Maybe we can use the word "pre-created namespace" instead.